.flowconfig

```diff
@@ -43,4 +43,4 @@ unsafe-getters-setters
 all=warn
 
 [version]
-^0.124.0
+^0.124.0
```

Comment:
you'll probably need to change this to `0.126.0`

also you should configure your editor to add a newline at the end of the file
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files#:~:text=Additionally%2C%20in%20the%20Settings%20page,there%20isn't%20already%20one.

Reply:
I'll address both of these now.